src/main/c/Objects/pyjtype.c

```diff
@@ -49,7 +43,7 @@ static int staticTypesInitialized = 0;
 static PyTypeObject* addCustomTypeToTypeDict(JNIEnv *env, PyObject* fqnToPyType,
         jclass class, PyTypeObject *type)
 {
-    if (PyDict_SetItemString(fqnToPyType, type->tp_name, (PyObject*) type)) {
+    if (PyDict_SetItemString(fqnToPyType, type->tp_name, (PyObject * ) type)) {
```

Comment:
Is there a reason there are spaces around the * now?  You did it multiple times in this file.

Reply:
astyle automatically added that in. I am using the same astyle configuration I have been using for years that I think I originally got from you but it is a newer version of astyle so it might have a slightly different default. I like the old way better so I will look into it and see if that is something I can change in the astyle configuration.

Reply:
This seems to be a bug with astyle 3.4.13, I've also tried 3.6.6 and I see the same behavior. Reverting to astyle 3.3 produces better results so I am going with that for this change.